velox/exec/Operator.cpp

```diff
@@ -361,4 +361,47 @@ void OperatorStats::clear() {
   runtimeStats.clear();
 }
 
+namespace {
+bool tryReserveAndRun(
```

Comment:
Not sure if we need this abstraction. But reserveAndRun can be in a separate PR if we need in production code? We can implement this method in DriverTest for test purpose?

Reply:
@oerling @xiaoxmeng has a good point.  Looks like we don't need to add reserveAndRun method to the Operator interface, do we?